trestlebot/entrypoints/autosync.py

```diff
@@ -90,69 +90,67 @@ def setup_autosync_arguments(self) -> None:
             help="Path to ssp index file",
         )
 
+    def validate_args(self, args: argparse.Namespace) -> None:
+        """Validate the arguments for the autosync entrypoint."""
+        authored_list: List[str] = [model.value for model in types.AuthoredType]
+        if args.oscal_model not in authored_list:
+            logger.error(
+                f"Invalid value {args.oscal_model} for oscal model. "
+                f"Please use one of {authored_list}"
+            )
+            sys.exit(const.ERROR_EXIT_CODE)
```

Comment:
You might consider defining exceptions which you could raise ... this would also reduce error exit handling to a single place.

Reply:
Thanks for this feedback. There is a `handle_exception` function being used in the entrypoint base for the core logic, but it does not include any of the input validation. I think that would be helpful. Addressing this in a follow-on PR - https://github.com/RedHatProductSecurity/trestle-bot/pull/103.